### ISSUE TITLE
I can't download clip menu from its web page

### ISSUE THREAD

**Issue by GuilleEneas** [ClipMenu/ClipMenu#16]

**Comment by igauravsehrawat** [comment]
Seems like dropbox file is inaccessible? A Github download link will help.

**Comment by pratheekhegde** [comment]
The download links are broken. 😥

**Comment by onurgozupek** [comment]
Any chance to reenable Dropbox links? We can not download files from clipmenu.com anymore...

**Comment by igauravsehrawat** [comment]
https://github.com/naotaka/ClipMenu Is this shifted here?

**Comment by CharlesMason** [comment]
FYI... I had and uploaded both the last beta ClipMenu_0.4.4a12.zip https://www.dropbox.com/s/w6ci4jtnbe3p0al/ClipMenu_0.4.4a12.zip?dl=0 I'm not sure what happen to the dev in Japan, he literally stop all internet activity around Nov. 2014... Nothing after that.  This works in Mac OS 10.11 (El Captain) and 10.12 (Sierra), I haven't test in 10.13 (High Sierra).

**Comment by CharlesMason** [comment]
@igauravsehrawat yes, he did shift it and was re-coding it completely using X-Code but as mentioned, something happened because there is ZERO internet activity after Nov. 2014.  Someone followed and copied called Clipy but I'm a little concerned because Little Snitch caught is "phoning home" and not sure what is was communicating but considering everything we copy-paste, I would not want those items getting to anyone else.

**Comment by onurgozupek** [comment]
Thank you @CharlesMason I guess there is a fork named Clippy and it works well.

**Comment by eskhool** [comment]
I am a little peeved that it doesn't declare that its a fork of ClipMenu loud and clear up front. Also its not a 'fork' as per github which is also not best ethically given he is riding on the shoulders of our Japanese 'giant' :)...best to him for a tool I have used for years

**Comment by bobinson** [comment]
Here is a version from Way back machine : https://web.archive.org/web/20160616151931/https://dl.dropboxusercontent.com/u/1140644/clipmenu/ClipMenu-1.0.0a1.dmg

**Comment by CharlesMason** [comment]
Bobinson, Thank you!! Yes that was the Alpha of the Cocoa version.  I have it but it never went beyond a1 and is not reliable compared to the 0.4.4a12 version. Which work in High  Sierra just fine.

**Comment by bobinson** [comment]
@CharlesMason - I was not aware of the reliability issue. I lost older version 0.4.x after a system crash and found this today. I have used it for last 10 hours without errors.

**Comment by CharlesMason** [comment]
@bobinson I had issues with certain items on copy-paste, I can't exactly remember but I want to say while in a PDF, but it would crash.  I'm using El Captain, so a newer OS might actually be better considering that the 1.0.0a1 was a start of a re-write in Cocoa.  I would be interested in seeing how 1.0.0a1 performs in High Sierra.

**Comment by bobinson** [comment]
@CharlesMason check #17

**Comment by CharlesMason** [comment]
@bobinson you should be able to download the 0.4.4a12 via the link I left from my Dropbox. If you need it sent direct I can do that.

**Comment by Smosker** [comment]
still cant download from web page

**Comment by onurgozupek** [comment]
> still cant download from web page

You can use the fork Clipy. https://github.com/Clipy/Clipy

**Comment by nishanthb** [comment]
> Here is a version from Way back machine : https://web.archive.org/web/20160616151931/https://dl.dropboxusercontent.com/u/1140644/clipmenu/ClipMenu-1.0.0a1.dmg

thanks buddy!

**Comment by nishanthb** [comment]
It would be nice to have a homebrew cask.

**Comment by bobinson** [comment]
> > Here is a version from Way back machine : https://web.archive.org/web/20160616151931/https://dl.dropboxusercontent.com/u/1140644/clipmenu/ClipMenu-1.0.0a1.dmg
> 
> thanks buddy!

:)

There is a similar project here https://github.com/Clipy/Clipy

**Comment by CharlesMason** [comment]
This is the last beta and working of ClipMenu_0.4.4a12.zip https://www.dropbox.com/s/w6ci4jtnbe3p0al/ClipMenu_0.4.4a12.zip?dl=0  This works in Mac OS 10.11, 10.12, 10.13, and 10.14.  I have verified these are working fine.  A direct download URL is https://uc6c2e3b453c61d1b33764dc58d2.dl.dropboxusercontent.com/cd/0/get/BIKm85gdpEzDUk7G37Sel1dA6iUAbBsiKdegFylgrtJJ8XMCziKZypL3GwSSSO891PauLjgTmwSfsB5u8FvmDV7kMnlsYnKn1Ipd58JDmBz_HA/file?_download_id=9997519188482292755179567725340879487871670371577160409560842808

I DO NOT RECOMMEND the ClipMenu-1.0.0a1.dmg as it was only a first time ALPHA (pre beta) and recommend the last stable and working version, ClipMenu_0.4.4a12 that works great up to 10.14.

Clipy is a branch of ClipMenu but I've had it "phone home", ie connect to the interest to report information back to an outside server, which is a MAJOR security concern. However, it has been a while since I've worked with it, being that ClipMenu works perfect on 10.14.

**Comment by SeaMac** [comment]
> @bobinson I had issues with certain items on copy-paste, I can't exactly remember but I want to say while in a PDF, but it would crash. I'm using El Captain, so a newer OS might actually be better considering that the 1.0.0a1 was a start of a re-write in Cocoa. I would be interested in seeing how 1.0.0a1 performs in High Sierra.

It works great in High Sierra! It has a search function, also. 
I wonder where the original Dev of this great project went? 
(The Wayback Machine extension told me where the Alpha 1.0.0 was stashed ... it has 2 copies!!! https://addons.mozilla.org/en-US/firefox/addon/wayback-machine_new/ )
